test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -490,34 +490,20 @@ def run_word_language_model(self, model_name):
         # Only support CPU version, since tracer is not working in GPU RNN.
         self.run_test(model, (x, model.hidden))
 
-    def get_image_from_url(self, url, size=(300, 200)):
+    def get_image(self, basename):
```

Comment:
We need this test in sync with torchvision unit tests. So we need a resizing at the end, maybe API should be
`def get_image(self, basename, size=(300, 200))` ?

Reply:
The files I'm checking in here are the resized versions, so the input tensors should be identical to what this test was producing before.

Can you please elaborate what "in sync" with means in this context and why we need that?

Reply:
"in sync" means any torchvision unit test should also be in pytorch unit test. Because torchvision updates pytorch once a while, if one torchvision unit test is not in pytorch unit test, and gets break after updating pytorch commit, it is not easy to find the root cause (because there are so many pytorch commits stacked).

Reply:
@neginraoof explained that I should make the change to use the same image files in pytorch/vision test/test_onnx.py. 

Doing that here: https://github.com/pytorch/vision/pull/4176